files/common.sh

```diff
@@ -336,19 +337,28 @@ install_package() {
     _package_and_version="${_package}"
   fi
 
-  if exists 'dnf'; then
-    exec_and_capture dnf install -y "${_package_and_version}"
-  elif exists 'yum'; then
-    exec_and_capture yum install -y "${_package_and_version}"
-  elif exists 'zypper'; then
-    exec_and_capture zypper install -y "${_package_and_version}"
-  elif exists 'apt'; then
-    exec_and_capture apt install -y "${_package_and_version}"
-  elif exists 'apt-get'; then
-    exec_and_capture apt-get install -y "${_package_and_version}"
-  else
-    fail "Unable to install ${_package}. Neither dnf, yum, zypper, apt nor apt-get are installed."
-  fi
+  case ${_os_family} in
+    debian|ubuntu)
+      if exists 'apt'; then
+        exec_and_capture apt install -y "${_package_and_version}"
+      elif exists 'apt-get'; then
+        exec_and_capture apt-get install -y "${_package_and_version}"
```

Comment:
Should we default to apt-get since it's non-interactive? I don't think it matters too much, really.

Reply:
Hmm, quite possibly; I tend to put apt first in mind for my own usage, but you're right that it is not the case that apt-get is deprecated and apt meant to replace it. Is apt-get less interactive than apt?